upup/models/cloudup/resources/addons/networking.flannel/k8s-1.6.yaml

```diff
@@ -1,3 +1,45 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: flannel
+  labels:
+    role.kubernetes.io/networking: "1"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes/status
```

Comment:
Im open to create a pull request back to CoreOS. This is the RBAC file from their repo:
https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel-rbac.yml